### PR TITLE
2021 2

### PR DESCRIPTION
**Pull request by JunAishima**
changes from late in the last shutdown and during the last cycle, so these have mostly been tested on the beamlines already.

### CODE REVIEW

File: bin/MT_LoadCheck.py
Comment (RobertSchaffer1) [inline bin/MT_LoadCheck.py:29]: I'm confused - why are we comparing a float and an int?

**Review by JunAishima — COMMENTED** [review]
Robert, I fixed the type issue in the MT_LoadCheck.py script with commit https://github.com/NSLS-II/lsdc/pull/104/commits/22dd77d7af6d15a3f31b47367f290f0927851086

File: bin/MT_LoadCheck.py
Comment (RobertSchaffer1) [inline bin/MT_LoadCheck.py:29]: Better, but is it still possible that this will round down (ie int takes 1.9999994 ->1 instead of 2)?

File: epics_det.py
Comment (RobertSchaffer1) [inline epics_det.py:199]: Maybe add "Timed out" or some such?

File: bin/MT_LoadCheck.py
Comment (JunAishima) [reply bin/MT_LoadCheck.py:29]: we are getting back a rounded floating point value from the underlying database as a string - so "2.50" or "2.0". I agree that this handling isn't ideal, but it is probably the best we can do with a general mechanism of returning numbers and imprecise floating point number formats.

File: epics_det.py
Comment (JunAishima) [reply epics_det.py]: agree with this - fixed with commit https://github.com/NSLS-II/lsdc/pull/104/commits/a98e6cfa78b50173f1b72839359844628d3090ec

File: lsdcGui.py
Comment (RobertSchaffer1) [inline lsdcGui.py:3144]: Consider adding 

`print("It's over 9000!") `

Comment (JunAishima) [reply lsdcGui.py:3144]: this is a conservative change that preserves the previous behavior. I won't change it now, but will consult with FMX to see what feedback users should get when the beam size is no longer changeable.

**Review by RobertSchaffer1 — APPROVED** [review]
I think it looks alright, there's a few style things that we could sort out later but otherwise I didn't notice issues. That said, it's all complicated enough that I have no real idea what other problems might arise - best to merge and test, I think. Wait for the second review though.

File: bin/MT_LoadCheck.py
Comment (RobertSchaffer1) [inline bin/MT_LoadCheck.py:20]: I think these are caught right now by the try statement - add an except clause sending them to errout?

File: bin/MT_LoadCheck.py
Comment (RobertSchaffer1) [inline bin/MT_LoadCheck.py:47]: Blank space at end of file

**Review by mskinner5278 — APPROVED** [review]

**Pull request merged**
